nipype/interfaces/base/tests/test_core.py

```diff
@@ -245,15 +245,17 @@ class input_spec(nib.TraitedSpec):
 
         _version = "misparsed-garbage"
 
-    obj = DerivedInterface()
+    with caplog.at_level(logging.WARNING, logger="nipype.interface"):
+        obj = DerivedInterface()
+    assert len(caplog.records) == 2
     obj.inputs.foo = 1
     obj.inputs.bar = 1
     with caplog.at_level(logging.WARNING, logger="nipype.interface"):
         obj._check_version_requirements(obj.inputs)
-    assert len(caplog.records) == 2
+    assert len(caplog.records) == 4
```

Comment:
This changed the expected warnings... I'm not sure if we want them at init?

Reply:
we could add a flag to suppress them in init.